### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by DannyShopgate**
# Adding Unit Tests

## Type of change

- [x] New feature

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
  All notable changes to this project will be documented in this file.
  The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+### [1.1.1] - 2019-03-26
```
Comment (sznowicki) [inline CHANGELOG.md]: What's the reason of releasing this? Wouldn't be enough to keep those changes unreleased? There's no code change except of one component renaming.

File: extension-config.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.1.0",
+  "version": "1.1.1",
```
Comment (sznowicki) [inline extension-config.json]: As above.

File: frontend/config.json [hunk 3]
```diff
@@ -1,49 +1,4 @@
 {
```
Comment (sznowicki) [inline frontend/config.json]: Please remove this file from git and add it to gitignore.

File: frontend/portals/CartItemListAfter/index.jsx [hunk 4]
```diff
@@ -5,7 +5,7 @@ import getConfig from '../../helpers/getConfig';
 export default () => {
   const config = getConfig();
 
-  if (config.checkboxValues.length === 0) {
+  if (!config.checkboxValues || config.checkboxValues.length === 0) {
```
Comment (sznowicki) [inline frontend/portals/CartItemListAfter/index.jsx]: Why this change is needed?

File: CHANGELOG.md (hunk 1 above)
Comment (DannyShopgate) [reply CHANGELOG.md]: That makes sense to me. I will remove

File: extension-config.json (hunk 2 above)
Comment (DannyShopgate) [reply extension-config.json]: Updated

File: frontend/portals/CartItemListAfter/index.jsx (hunk 4 above)
Comment (DannyShopgate) [reply frontend/portals/CartItemListAfter/index.jsx]: Leftover implementation for a test. Removed because I mock config in test now.

File: frontend/config.json (hunk 3 above)
Comment (DannyShopgate) [reply frontend/config.json]: frontend/config.json in gitignore file

Comment (DannyShopgate) [reply frontend/config.json]: removed file

File: frontend/portals/CartItemListAfter/index.jsx (hunk 4 above)
Comment (DannyShopgate) [reply frontend/portals/CartItemListAfter/index.jsx]: Also removed unit test where I mocked null config.

File: frontend/package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shopgate/terms-and-conditions",
-  "version": "1.1.0",
+  "version": "1.1.1",
```
Comment (sznowicki) [inline frontend/package.json]: This is still 1.1.1. Please roll it back to 1.1.0.

Comment (DannyShopgate) [reply frontend/package.json]: ah shoot. updated
